charts/allure/templates/service-ui.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "allure.fullname" . }}-ui
+  labels:
+    {{- include "allure-ui.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
Optional... I think it may also be good setting service type independent for UI and API. I would suggest refactoring values to include two root sections `.Values.api` and `.Values.ui` having nested attributes in these sections